modules/installation-configuration-parameters.adoc

```diff
@@ -1073,6 +1073,10 @@ Additional Azure configuration parameters are described in the following table:
 |Optional. The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt compute machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
 |String, in the format `00000000-0000-0000-0000-000000000000`.
 
+|`compute.platform.azure.vmNetworkingType`
+|Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance. If instance type of compute machines support `Accelerated` networking, installer by default enables `Accelerated` networking, otherwise the default networking type is `Basic`.
```

Comment:
```suggestion
|Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance. If instance type of compute machines support `Accelerated` networking, by default, the installer enables `Accelerated` networking, otherwise the default networking type is `Basic`.
```

Reply:
Also the `If instance type of compute machines support...` statement is mentioned in `Values` column for the other 2 fields. can you make this consistent?